sympy/printing/tests/test_mathml.py

```diff
@@ -3,6 +3,9 @@
     pi, GoldenRatio, EulerGamma, Sum, Eq, Ne, Ge, Lt, Float, Matrix, Basic, \
     S, MatrixSymbol, Function, Derivative, log, true, false, Range, Min, Max, \
     Lambda, IndexedBase, symbols, MatrixSymbol
+from sympy import elliptic_k, totient, reduced_totient, primenu, primeomega, \
+    fresnelc, fresnels, Heaviside
+from sympy.calculus.util import AccumBounds
```

Comment:
These are probably duplicated (since they did not affect your test they shouldn't be added, but I know I added them recently so probably this is a result of a resolved merge conflict).

Reply:
I see the MatrixSymbol being duplicated. That's the only thing I should remove right? or does the code in green has some duplicates as well?